.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Lets add this file to gitignore :)

Reply:
You mean you don't keep your SDK at this location? 😄

I carefully skipped this on every change, until I stashed to merge and then brought it back 🤦‍♂️ Fixed!